ibm/mas_devops/roles/db2_backup/defaults/main.yml

```diff
@@ -2,6 +2,7 @@
 # Db2 backup default vars
 db2_backup_dir: "{{ lookup('env', 'DB2_BACKUP_DIR') }}"
 db2_backup_instance_name: "{{ lookup('env', 'DB2_BACKUP_INSTANCE_NAME') }}"
+db2_namespace: "{{ lookup('env', 'DB2_NAMESPACE') }}"
```

Comment:
@cuddlyporcupine  To be consistent with default values used in db2 role, I would suggest we default namespace in backup/restore roles with same value as [here](https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/db2/defaults/main.yml#L4)

Reply:
@alequint requested change for default is in